DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetLobby.cs

```diff
@@ -406,6 +435,70 @@ private void PostUIInit()
             GameProcessLogic.GameProcessExited += SharedUILogic_GameProcessExited;
         }
 
+        private void AccountLoginWindow_Cancel(object obj)
+        {
+            UpdateAccountLoginState();
+        }
+
+        /// <summary>
+        /// Login with nickname picked from account
+        /// </summary>
+        /// <param name="obj"></param>
```

Comment:
Avoid empty param comments. If you don't think it needs to be explained, just remove the whole param comment line.